README.md

```diff
@@ -1,358 +1,48 @@
 
-    if (listPushresult is int) {
-        io:println(io:sprintf("Number of elements pushed: %s", listPushresult));
-    } else {
-        io:println("Error occurred while calling `lPush`");
-    }
+[Redis](https://redis.io/) is an open source, in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets.
 
-    io:println("Poping an element from NumberList");
-    var lPopResult = conn->lPop("NumberList");
+This connector provides operations for connecting and interacting with Redis server over the network. 
+For more information about configuration and operations, go to the module. 
+- [`ballerinax/redis`](redis/Module.md)
 
-    if (lPopResult is string) {
-        io:println("Popped element: " + lPopResult);
-    } else if (result is ()) {
-        io:println("Key not found");
-    } else {
-        io:println("Error occurred while calling `lPop`");
-    }
-    io:println("===Executing sample set oprerations===");
-    //Sample Set Operations
-    io:println("Adding 3 elements to NumberSet");
-    var setAddResult = conn->sAdd("NumberSet", ["1", "2", "3"]);
+# Building from the Source
+## Setting Up the Prerequisites
 
-    if (setAddResult is int) {
-        io:println(io:sprintf("Number of elements added: : %s", setAddResult));
-    } else {
-        io:println("Error occurred while calling `sAdd`");
-    }
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
 
-    io:println("\nQuerying number of elemenets in the Set");
-    var setCardResult = conn->sCard("NumberSet");
+   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
 
-    if (setCardResult is int) {
-        io:println(io:sprintf("Number of memebers in the set: %s", setCardResult));
-    } else {
-        io:println("Error occurred while calling `sCard`");
-    }
+   * [OpenJDK](https://adoptopenjdk.net/)
 
-    io:println("===Executing sample hash oprerations===");
-    //Sample Hash operations
-    io:println("Adding a key value pair to a hash");
-    var hashSetResult = conn->hSet("HashKey", "Name", "Manuri");
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
 
-    if (hashSetResult is boolean) {
-        io:println(io:sprintf("Hash set status: %s", hashSetResult));
-    } else {
-        io:println("Error occurred while calling `hSet`");
-    }
+2. Download and install [Ballerina Swan Lake Beta2](https://ballerina.io/). 
 
-    io:println("Querying the value of the hash field Name");
-    var hashGetResult = conn->hGet("HashKey", "Name");
+## Building the Source
+Execute the commands below to build from the source.
 
-    if (hashGetResult is string) {
-        io:println("Value of the hash field: " + hashGetResult);
-    } else {
-        io:println("Error occurred while calling `hGet`");
-    }
+1. To build the package:
```

Comment:
Suggestion:
Let's use bullet points instead of numbers because users don't need to perform the steps in a sequence.

Reply:
Yes, I will change. That is more meaningful

Reply:
As we have a common format across all the connectors. If we change this it will be inconsistent when compared with other connectors

Reply:
Changed according with above suggestion